lib/src/executable/watch.dart

```diff
@@ -288,8 +288,8 @@ class _Watcher {
     var changed = <StylesheetNode>[];
     for (var node in _graph.nodes.values) {
       var importChanged = false;
-      for (var url in node.upstream.keys) {
-        if (_name(p.url.basename(url.path)) != name) continue;
+      void recanonicalize(Uri url, StylesheetNode upstream, bool forImport) {
```

Comment:
Nit:

```suggestion
      void recanonicalize(Uri url, StylesheetNode upstream, {@required bool forImport}) {
```

This just helps make it clearer at the callsite what the boolean means.